webapp/view/ListReport.view.xml

```diff
@@ -0,0 +1,187 @@
+<mvc:View
+	xmlns="sap.m"
+	xmlns:mvc="sap.ui.core.mvc"
+	xmlns:f="sap.f"
+	xmlns:core="sap.ui.core"
+	xmlns:fb="sap.ui.comp.filterbar"
+	controllerName="products.app.controller.ListReport">
+
+		<f:DynamicPage 
+			id="dynamicPageId" 
+			preserveHeaderStateOnScroll="true" 
+			showFooter="true" 
+			fitContent="true">
+			<!-- DynamicPage Title -->
+			<f:title>
+				<f:DynamicPageTitle>
+					<f:heading>
+						<Title text="{i18n>ListReportTitle}"/>
+					</f:heading>
+					
+				</f:DynamicPageTitle>
+			</f:title>
+
+			<!-- DynamicPage Header -->
+			<f:header>
+				<f:DynamicPageHeader>
+					
+					<f:content>
+						
+						<fb:FilterBar id="filterbar" persistencyKey="myPersKey" useToolbar="false" >
+							<fb:filterGroupItems >
+
+								<fb:FilterGroupItem 
+									name="search" 
+									label="{i18n>NameText} {i18n>AndText} {i18n>DescriptionText}" 
+									groupTitle="false" 
+									groupName="HeaderFiltersGroup" 
+									visibleInFilterBar="true">
+									<fb:control>
+										<SearchField 
+											id="searcher" 
+											width="100%" 
+											liveChange="onFilter(null)" 	
+										/>
+									</fb:control>
+								</fb:FilterGroupItem>
+
+								<fb:FilterGroupItem 
+									name="Availability" 
+									label="{i18n>AvailabilityText}" 
+									groupName="HeaderFiltersGroup" 
+									visibleInFilterBar="true">
+									<fb:control>
+										<Select
+											id="availabilitySelect"
+											forceSelection="false"
+											selectedKey="all"
+											change="onFilter(null)"
+										>
+										<core:Item key="all" text="all" />
+										<core:Item key="false" text="In Stock" />
+										<core:Item key="true" text="Out of Stock" />
+									</Select>
+									</fb:control>
+								</fb:FilterGroupItem>
+
+								<fb:FilterGroupItem 
+									name="Price range" 
```

Comment:
No hardcoded text.
You could possibly think that it is not displayed in the UI so it doesn't need i18n. But this is partially true.
Your application can be used by people who have vision disorders, so they will use tools like screen readers which will tell them what they are interacting with.

In example, a blind from Germany uses your app. He wants to filter products by price, but what he hears is "Price range", not german "Preisklasse". He confused.

Please use i18n here and everywhere else

Reply:
But we can't to add in "name" text from i18n like this: name="{i18n>PriceRangeText}" , or with parts and formatter. It doesn't work for property name

Reply:
Interesting. Will check. For now just leave it